Apps/APN/package.json

```diff
@@ -11,8 +11,8 @@
     "clean-install-ios": "./scripts/clean-run-ios.sh",
     "dev:update": "../../scripts/dev:update.sh",
     "postinstall": "pod update --project-directory=ios",
-    "preinstall": "npm run dev:update --legacy-peer-deps",
-    "ci:install": "npm run preinstall && npm ci --legacy-peer-deps && npm run bundle:ios"
+    "preinstall": "npm run dev:update",
+    "ci:install": "npm run preinstall && npm ci"
```

Comment:
think they are not needed due to the react native version?

Reply:
Yeah, we now user newer versions so longer need to install legacy dependencies